website/content/api-docs/relatedtools.mdx

```diff
@@ -6,12 +6,13 @@ description: Short list of third-party tools that work with or are related to Va
 
 # Related Tools
 
-## Hashicorp Tools
+## HashiCorp Tools
 
-- The [Terraform Vault provider](https://registry.terraform.io/providers/hashicorp/vault/latest/docs) can read from, write to, and configure Vault from [HashiCorp Terraform](https://www.terraform.io/)
+- [Vault Agent](/vault/docs/agent-and-proxy/agent) can render Vault secrets either to files or directly into a child process using `consul-template` templating syntax
```

Comment:
`or directly into a child process` -> `or directly into a child process as environment variables`

Just to make it clear on the mechanism at play :)

Reply:
Good point 👍